src/main.rs

```diff
@@ -463,6 +472,7 @@ mod tests {
                 room: -1234567890i64
             }),
             run_interval: 999,
+            cron_interval: "59 23 * * *".to_owned(),
```

Comment:
Will do, I just need confirmation that the cron code works as it should because I'm not too sure how to use this code personally myself.